Casks/l/loungy.rb

```diff
@@ -0,0 +1,22 @@
+cask "loungy" do
+  version "0.1.2"
+  sha256 "81700f11ed264fceae4623b906de98b52340f760baa1dede3ff5c2eeaf008320"
+
+  url "https://github.com/MatthiasGrandl/Loungy/releases/download/v#{version}/Loungy_#{version}_universal.dmg"
+  name "Loungy"
+  desc "GPU accelerated application launcher in the vein of Raycast and Alfred"
```

Comment:
```suggestion
  desc "Application launcher"
```

Feel free to suggest an alternative to this, but make sure it meets the [`desc guidelines`](https://docs.brew.sh/Cask-Cookbook#stanza-desc).